docs/features/parameters.md

```diff
@@ -43,7 +43,41 @@ public class ReportJob : IJob
 }
 ```
 
+It's also possible to configure a job with only a parameter and no cron expression.
+
+```csharp
+Services.AddNCronJob(options =>
+{
+    options.AddJob<MaintenanceJob>(j =>
+    {
+        j.WithParameter("lightMode");
+    });
+});
+```
+
+This can later be triggered through the `IInstantJobRegistry` in a preconfigured mode (See [_"Instant jobs"_](./instant-jobs.md)).
+
+```csharp
+app.MapPost("/maintenance/light", (IInstantJobRegistry jobRegistry) =>
+{
+    // "lightMode" will be passed as a parameter to the job
+    jobRegistry.RunInstantJob<MaintenanceJob>();
```

Comment:
But we don't have a way of having "nothing" as parameter.

Reply:
Test https://github.com/NCronJob-Dev/NCronJob/blob/e93ccfead09ed521cdf67086bf249a8962d8861d/tests/NCronJob.Tests/RuntimeJobRegistryTests.cs#L312 has been added in #249

Tracked through #251